src/v2/Components/ArtworkFilter/__tests__/FiltersPills.jest.tsx

```diff
@@ -0,0 +1,46 @@
+import { mount } from "enzyme"
```

Comment:
**suggestion**: use `@testing-library/react` instead of `enzyme`

Reply:
Recently updated docs: https://github.com/artsy/force/blob/master/docs/best_practices.md#write-unit-tests-for-new-components